src/components/Checkbox/src/stories/Checkbox.stories.mdx

```diff
@@ -0,0 +1,399 @@
+import { ArgsTable, Canvas, Description, Meta, Story } from "@storybook/addon-docs";
+import { DesignTokenDocBlock } from "storybook-design-token/dist/doc-blocks";
+import { action } from "@storybook/addon-actions";
+
+import { Checkbox } from "..";
+import { FormControlLabel } from "../../../FormControlLabel";
+import { FormGroup } from "../../../FormGroup";
+import pkg from "../../package.json";
+import readme from "../../README.md";
+
+<Meta
+  title="Components/Input/Checkbox"
+  component={Checkbox}
+  parameters={{ componentSubtitle: `${pkg.name} - ${pkg.version}`, docs: { source: { type: "dynamic" } } }}
+/>
+
+<Description markdown={readme} />
+
+## Design Tokens
+
+<DesignTokenDocBlock categoryName="Checkbox" viewType="table" />
+
+## Stories
+
+### Checkbox only
+
+<Canvas>
+  <Story
+    name="Without FormControlLabel"
+    args={{}}
+    parameters={{
+      docs: {
+        source: {
+          code: `
+import { Checkbox } from "@gemeente-denhaag/checkbox";
+const ReactCheckBox = <Checkbox onChange={() => {}}></Checkbox>;
+// This 👆 transpiles into this 👇 in the DOM
+<span class="denhaag-checkbox">
+  <input type="checkbox" class="denhaag-checkbox__input">
+  <div class="denhaag-checkbox__icon">
+    <svg ...> ...</svg>
+  </div>
+</span>
+`,
+        },
+      },
+    }}
+  >
+    {(args) => <Checkbox onChange={action("Checkbox action")}></Checkbox>}
+  </Story>
+</Canvas>
+
+### With FormControlLabels
+
+<Canvas>
+  <Story
+    name="Default"
+    args={{}}
+    parameters={{
+      docs: {
+        source: {
+          code: `
+<FormControlLabel
+  label="Checkbox"
+  control={<Checkbox onChange={() => {}}></Checkbox>}
+></FormControlLabel>
+// This 👆 transpiles into this 👇 in the DOM
+<label class="denhaag-form-control-label">
```

Comment:
We could probably remove the second part since we also have a pure html example further down. What are your thoughts on this @bddjong>

Reply:
Agreed